.github/CODEOWNERS

```diff
@@ -0,0 +1,104 @@
+# This file describes the maintainers for the main components. See
+# `dev/doc/MERGING.md`.
+
+/.github/         @maximedenes @Zimmi48
+/dev/doc/         @Zimmi48 @maximedenes
+
+/checker/         @barras @maximedenes
+
+# We need a secondary maintainer for clib
+/clib/            @ppedrot
+
+/doc/             @silene @maximedenes
+/man/             @silene @maximedenes
+
+/engine/          @ppedrot @aspiwack
+
+/grammar/         @ppedrot @maximedenes
+
+/ide/             @ppedrot @gares
+
+# We need a secondary maintainer for interp
+/interp/          @herbelin
+
+/intf/            @letouzey @ppedrot
+/kernel/          @maximedenes @barras
+/lib/             @ejgallego @ppedrot
+/library/         @silene @gares
+
+/parsing/         @herbelin @mattam82
+
+# We need secondary maintainers for many of these
+/plugins/btauto/  @ppedrot
+
+/plugins/cc/            @herbelin # I don't know Pierre Corbineau's GitHub nickname
+/plugins/derive/        @aspiwack @ppedrot
+/plugins/extraction/    @letouzey @maximedenes
+/plugins/firstorder/    @herbelin # I don't know Pierre Corbineau's GitHub nickname
+/plugins/fourier/       @herbelin @gares
+/plugins/funind/        @forestjulien @Matafou
+/plugins/ltac/          @ppedrot
+/plugins/micromega/     @fajb @bgregoir
+/plugins/nsatz/         @thery
+/plugins/omega/         @letouzey
+/plugins/romega/        @letouzey
+/plugins/setoid_ring/   @amahboubi @bgregoir
+/plugins/ssrmatching/   @gares @maximedenes
+/plugins/ssr/           @gares @maximedenes
+/plugins/syntax/        @ppedrot @maximedenes
+
+
+# We need a maintainer for quote
+#/plugins/quote/
+
+# Should be Pierre Corbineau and someone else
+#/plugins/rtauto
+
+/pretyping/       @mattam82 @gares
+/printing/        @herbelin @mattam82
+
+# We need a secondary maintainer for proofs
+/proofs/          @ppedrot @Zimmi48
+
+/stm/             @gares @ejgallego
+
+/tactics/         @ppedrot @mattam82
+
+# We need a secondary maintainer for many of these
+/theories/Arith/          @letouzey
+/theories/Bool/           @letouzey
+/theories/Classes/        @mattam82
+/theories/FSets/          @letouzey
+/theories/Init/           @letouzey @ppedrot
+/theories/Lists/          @letouzey @ppedrot
+/theories/Logic/          @herbelin @ppedrot
+/theories/MSets/          @letouzey
+/theories/NArith/         @letouzey
+/theories/Numbers/        @letouzey
+/theories/PArith/         @letouzey
+/theories/Program/        @mattam82
+/theories/QArith/         @letouzey
+/theories/Reals/          @silene @ppedrot
+/theories/Relations/      @mattam82 @ppedrot
+/theories/Setoids/        @mattam82 @ppedrot
+/theories/Sets/           @letouzey
+/theories/Sorting/        @letouzey
+/theories/Strings/        @letouzey @herbelin
+/theories/Structures/     @letouzey
+/theories/Unicode/        @letouzey @herbelin
+/theories/Wellfounded/    @letouzey @mattam82
+/theories/ZArith/         @letouzey
+
+# We need a maintainer for these
+#/theories/Compat/
```

Comment:
I suggest @JasonGross for `Compat/`

Reply:
Sure, if @JasonGross is ok. Note however that since he's the main contributor to those files, we'd need a secondary maintainer.

Reply:
Sure, though I don't think I have push bits for running the merge script, which according to `dev/doc/MERGING.md` would be my responsibility for changes primarily about `Compat` files?